sql/core/src/main/scala/org/apache/spark/sql/DataFrameNaFunctions.scala

```diff
@@ -394,10 +395,11 @@ final class DataFrameNaFunctions private[sql](df: DataFrame) {
   }
 
   private def fillMap(values: Seq[(String, Any)]): DataFrame = {
+    val resolved = mutable.Map[String, Any]()
```

Comment:
instead of having mutable map, you can just do the following:
```scala
val resolved = values.map { case (colName, replaceValue) =>
  ...
  ...
  (resolvedColumn.name, replaceValue)
}  
```

Reply:
Done.